src/parser/parse_tree_cl.c

```diff
@@ -9182,6 +9201,19 @@ pt_print_spec (PARSER_CONTEXT * parser, PT_NODE * p)
 	}
       r1 = pt_print_bytes (parser, p->info.spec.entity_name);
       q = pt_append_varchar (parser, q, r1);
+
+      if (p->info.spec.remote_server_name)
```

Comment:
This code should be removed.
It comes here only if it is PT_META_CLASS.

```
create table t1 class attribute (a1 int) (c1 int) dont_reuse_oid;

set optimization level 513;
select /*+ recompile */ * from class t1;
```
```
create table t2 (c1 int) dont_reuse_oid;
create table t3 as subclass of t2 (c1 int) dont_reuse_oid;

set optimization level 513;
select /*+ recompile */ * from all t3;
```

Reply:
fixed.